core_domain.process_model_execution/src/flow_node_handler/activity_handler/call_activity_handler.ts

```diff
@@ -239,9 +271,39 @@ export class CallActivityHandler extends ActivityHandler<Model.Activities.CallAc
 
     const result = await this
       .executeProcessService
-      .startAndAwaitEndEvent(identity, options);
+      .start(identity, options);
+
+    this.subProcessInstanceId = result.processInstanceId;
+
+    return new Promise((resolve, reject) => {
+      const processInstanceTerminated = eventAggregatorSettings.messagePaths.processInstanceWithIdTerminated.replace(
+        eventAggregatorSettings.messageParams.processInstanceId,
+        this.subProcessInstanceId,
+      );
+
+      this.eventAggregator.subscribe(processInstanceTerminated, (payload) => {
```

Comment:
Es wäre besser hier "subscribeOnce" zu nutzen, damit der Event Aggregator die Subscriptions automatisch abräumt.
Sonst müsstest du dich da selbst drum kümmern.

```suggestion
      this.eventAggregator.subscribeOnce(processInstanceTerminated, (payload) => {
```